Assets/Tests/InputSystem/Plugins/InputForUITests.cs

```diff
@@ -31,13 +31,12 @@ public override void Setup()
     {
         base.Setup();
 
-        // Test assumes a compatible project-wide action configuration exist for UI
-        // See CoreTests_ProjectWideActions for how its setup via build plugins.
-        Assert.That(InputSystem.actions, Is.Not.Null,
-            "Test is invalid since Project-wide Input System actions have not been setup for play-mode test environment");
-
         m_InputSystemProvider = new InputSystemProvider();
         EventProvider.SetMockProvider(m_InputSystemProvider);
+
+        // Test assumes a compatible action asset configuration exists for UI
```

Comment:
I would suggest to not have asserts in Setup and only have asserts to actually verify functional requirements as part of test methods. For what scenario would this fail were it wouldn't fail the functional tests anyway? Otherwise if it renders the test non-applicable use assumption failure or throw exception if programmer error?

Reply:
I've moved this check out to a separate test in 404f0a2b9aefb2d21e7e69038f0103a5f5167189.